docs: update documentation and definitions files (#99)
dashevo/DashJS: docs/platform/identities/topUp.md

**Usage**: `client.platform.identities.topUp(identity, amount)`    

> **Description**: This method will register a new identity for you. 

Reviewer: ```suggestion
**Description**: This method will topup the provided identity's balance. 
```
